src/chapters/arrays/array-methods.rst

```diff
@@ -0,0 +1,151 @@
+.. _array-methods:
+
+Array Methods
+==============
+
+.. index::
+   single: array; methods
+
+As with strings, JavaScript provides us with useful **methods** for arrays.
+Unlike strings, arrays are *mutable* and can be changed.
+
+Some methods alter an array, while others *return* values. For example,
+``console.log([1, 2, 3].includes(2))`` prints ``true``. The  action
+``[1, 2, 3].includes(2)`` checks the array to see if it contains the item ``2``
+and then *returns* a boolean which can be printed (or assigned to a variable).
+
+Common Array Methods
+--------------------
+
+Here is a sample of the most frequently used array methods. More complete lists
+can be found here:
+
+#. Novice list: `W3 Schools Array Methods <https://www.w3schools.com/jsref/jsref_obj_array.asp>`__
```

Comment:
Maybe don't distinguish this as "novice". Something like "Quick overview".

Reply:
Ok.  Change made.